articles/applications/how-to-manage-repositories/how-to-manage-repositories.md

```diff
@@ -32,8 +32,10 @@ Click on `Settings` and then on `Manage Repositories`.
 
 See [here](xref:user-interface.about-asset-repositories) on how the Asset Management dialog works.
 
->[!NOTE]
->A `intent.repositories.config` file will be created in the same folder as your `.isln` file for Intent Architect. Deleting this file will reset behaviour so that Intent Architect will scan your global level repositories again.
+> [!NOTE]
+> A `intent.repositories.config` file will be created in the same folder as your `.isln` file for Intent Architect. Deleting this file will reset behaviour so that Intent Architect will scan your global level repositories again.
+> [!NOTE]
+> In almost all cases [global level](#managing-on-a-global-level) repositories should be used. Solution level repositories should only be used when you want to use a relative path to assets which is consistent across different developer computers, such as when the modules solution is in the same source code repository as the solution which uses the modules.
```

Comment:
I feel this should rather be at the start of the section, that way people can know before reading the section that they probably don't want to use it.

We actually generally want to discourage use of this feature, barring maybe 1 of our clients, whenever its been set up, it was in error and caused problems for the client.